test/index.js

```diff
@@ -1,14 +1,10 @@
 'use strict';
 
-var test = require('tape');
-var isArrowFunction = require('../index');
-var arrowFuncs = require('make-arrow-function').list().slice(0, -2);
-var asyncFuncs = require('make-async-function').list();
-var generatorFuncs = require('make-generator-function')();
-var forEach = require('for-each');
+const test = require('tape');
```

Comment:
similarly, we support engines that lack const, so we must always use var.

Reply:
I've avoided anything ES5 or up in `index.js`, can do the same for tests as needed. Does this mean the tests have the same ES3 support requirement though?

Reply:
Yes - how else can we verify the implementation works in those envs?